packages/react-divider/Spec.md

```diff
@@ -0,0 +1,137 @@
+
+[Github Epic Convergance Issue #16254](https://github.com/microsoft/fluentui/issues/16254)
+
+[Github Convergance Issue #15759](https://github.com/microsoft/fluentui/issues/15759)
+
+[Fluent UI Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+
+[Open UI Component Matrix](https://open-ui.org/analysis/component-matrix)
+
+[Ant Design Divider](https://ant.design/components/divider/)
+
+[Fast Divider](https://explore.fast.design/components/fast-divider)
+
+[Semantic UI Divider](https://semantic-ui.com/elements/divider.html)
+
+## Comparison of v8 and v0
+
+- v8 - [Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+- v0 - [Divider](https://fluentsite.z22.web.core.windows.net/0.51.4/components/divider/definition)
+
+## Sample Code
+
+Basic Examples:
+
+```html
+<Divider />
+<Divider vertical="true" />
+<Divider>This is a divider with content</Divider>
+```
+
+## Variants
+
+The divider will have two main variants, a horzontal and vertical render. The control is the same with the optional property `vertical="true"` defining the rendering type. As per current definition, content can only be added if the divider is horizontal (TBD).
+
+Standard default horizontal
+
+```html
+<Divider />
+```
+
+Vertical
+
+```html
+<Divider vertical="true" />
+```
+
+## API
+
+From [Divider.types.tsx](http://about:blank)
+
+### Slots
+
+- content (?)
+
+### Props
+
+```ts
+export type DividerProps = ComponentProps &
+  React.HTMLAttributes<HTMLElement> & {
+    /**
+     * A divider can justify the content. Center is default.
+     */
+    alignContent?: 'start' | 'end' | 'center';
+
+    children?: any;
+
+    /* A divider can have a overriding color */
+    color?: string;
+
+    /**
+     * Shorthand children content within the button.
+     */
+    content?: ShorthandProps;
+
+    /* A divider can remove any extra margins before or after the element */
+    fitted?: boolean;
+
+    /* important will emphasis the content */
+    important?: boolean;
+
+    /* The size is a multiplier value for the size */
+    size?: SizeValue;
+
+    tokens?: RecursivePartial<DividerTokens>;
+
+    /* A divider can be horizontal (default) or verticle*/
+    vertical?: boolean;
+  };
+```
+
+## Styling Tokens
+
+TBD
+
+```ts
+export type DividerTokens = ColorTokens & FontTokens & {};
+```
+
+## Structure
+
+<div class="root">
+  <!-- ::before to handle the divider line for with or without content -->
+  <div>Content</div>
+  <!-- ::after to handle the post divider line if content exists -->
+</div>
+
+## Migration
+
+The divider control is a direct replacement of the current Separator control, all current properties will remain intact, however the tag name will need to be updated from `<Separator/>` to `<Divider/>`
+
+Migration from V0 will require removing the content property and nesting that content as a child of the node ex.
+
+```
+<Divider content="sdfdsF">
+<Divider>sfdjkhjsdfhjsdf</Divider>
+
+```
```

Comment:
This seems to be different from the API and structure you outline above. What is the approach that we are taking?

Reply:
This was tbd .. has been updated to reflect accordingly